### ISSUE TITLE
Team invite to an RPE includes a link that causes a loop

### ISSUE THREAD

**Issue by dboyer** [Iridescent-CM/technovation-app#3690]
A ChA can invite specific teams to join an RPE on this admin page.

<img width="1017" alt="Screen Shot 2022-10-27 at 4 07 50 PM" src="https://user-images.githubusercontent.com/516764/198387937-ac158399-8d63-48b2-8b1d-3f63b0af74d2.png">

The team members receive an email that includes a button to "log in for more details." If you click this event, you get the error page and stuck in a loop where it is impossible to view any page on QA. 
<img width="607" alt="Screen Shot 2022-10-27 at 4 31 45 PM" src="https://user-images.githubusercontent.com/516764/198391946-3f44c20a-c609-4174-a891-c032c5e8151c.png">

The specific error is here:
<img width="1205" alt="Screen Shot 2022-10-21 at 11 17 34 AM" src="https://user-images.githubusercontent.com/516764/198392668-07fdb641-5261-4ced-bf9a-c3d7ae0bbe44.png">

This card should cover fixing the error so that we can see where the link in the email takes students.

**Comment by dboyer** [comment]
The link in the email may be trying to direct to /student/regional_pitch_events which also causes an error and gets the user stuck in a loop when trying to access it from the button at /student/dashboard#/events

**Comment by dboyer** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Technovation-605271851956b1000ec2f24d/issues/Iridescent-CM/technovation-app/3690?planning-poker) with Zenhub @shaunxp20 @viviancan

**Comment by shaunxp20** [comment]
@dboyer Are you by chance impersonating when running into this error?

When I was impersonating, I ran into an error (although it wasn't a polymorphic route error), but when i opened the link (in the invite email) in a fresh window/tab it worked as expected. I tried with three separate teams and it's working as expected. If it's still an issue, I'll need the steps and criteria, maybe I'm doing something different or it's a specific team or team member that's causing an issue? 🤔

**Comment by dboyer** [comment]
I get the error when I follow this series of events:

1. Login as the Deb Chile ChA account (deb+cha1028...)
2. Go to /chapter_ambassador/events#/ and select the Chile Chapter Celebration event. Click the "manage teams" icon and then the add teams button.
3. Select "maze madness" team and then click Done. Then click the "save selected teams" button.

Steps 1-3 are shown [in this recording](https://drive.google.com/file/d/1PkYgTDP4ATLPtkfTt6ODYfs9HdpTZgNE/view)

4. Logged out of the Deb Chile ChA account so I am logged into no account on QA.
5. Opened the email with the invite for the team to join the event
<img width="586" alt="Screen Shot 2022-11-07 at 10 11 51 AM" src="https://user-images.githubusercontent.com/516764/200345020-c5446cdf-dbed-4575-8eea-fc7d31f857db.png">

6. Clicked the Log In for More Details button and it opened a new tab with the error page. 

<img width="845" alt="Screen Shot 2022-11-07 at 10 12 29 AM" src="https://user-images.githubusercontent.com/516764/200345187-ec898c7d-c421-4191-ac33-4d6076cdded2.png">

7. I clicked the "go back to the dashboard" link on the error page and it loads the /student/dashboard but it still shows as an error. When I close that tab and open a new one and go to QA, I still get the error. The only way to get off the error page is by going to /logout to force the logout.

**Comment by dboyer** [comment]
Interestingly, I tried to login as that student (deb+senior0920@technovation) and it's still taking me to the error page even after forcing the logout. I tried logging in with a different account and it was fine.

**Comment by dboyer** [comment]
Oh that is infuriating. I double checked and wasn't logged in with another account or impersonating so I don't think that is the issue. Vivian saw this too so I swear I am not imagining it. I hope.....

**Comment by viviancan** [comment]
I just shared a screen recording of how I got caught in the loop!

**Comment by shaunxp20** [comment]
I was finally able to get to the bottom of this one! Thanks for your help @dboyer and @viviancan! Indeed it was the "RPE" setting under "Content & Settings" that was enabled on QA, but I didn't have it enabled locally - after I enabled, I ran into the same issue! I fixed it, so the link will actually work! Side note: it seems the link goes to the dashboard vs the actual RPE. 🤔 Anywho, it should on QA soon to see/play with more! 😄
